src/core/echarts.ts

```diff
@@ -1131,6 +1132,7 @@ class ECharts extends Eventful<ECEventDefinition> {
 
                     params.event = e;
                     params.type = eveName;
+                    params.dimensionIndex = dispatcher ? dispatcher.__dimIdx : undefined;
```

Comment:
If `__dimIdx` is only for radar series, then it's not a good idea to bind it in `core/echarts.ts` because it breaks the encapsulation. Find other ways to bypass the information with `eventData` may be a better idea.

Reply:
Done